### PR TITLE
Update Regex, Cache, File, Xmldata, Task and Xslt BBEs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: examples/directory-listener/directory_listener.bal [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-// In this example, the listener monitors any modifications done to a specific directory.
-// Before running the example, change the value of the 'path' field 
-// to indicate the path of the directory that you want the listener to monitor.
-// As the recursive property is set to false,
-// the listener does not monitor the child directories of the main directory
-// that it listens to.
+// The listener monitors any modifications done to a specific directory.
+// Before running this, change the value of the 'path' field to indicate the path of the
+// directory that you want to monitor. As the recursive property is set to false,
+// the listener does not monitor the child directories of the main directory that it listens to.
```
Comment (daneshk) [inline examples/directory-listener/directory_listener.bal:7]: Shall we move this to MD file

File: examples/cache-basics/cache_basics.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 # Cache basics
 
-The Ballerina Cache API provides an in-memory cache implementation by default with a
+The `cache` library provides an in-memory cache implementation APIs by default with the
```
Comment (daneshk) [inline examples/cache-basics/cache_basics.md:3]:
```suggestion
The `cache` library provides an in-memory cache implementation APIs with the
```

File: examples/cache-basics/cache_basics.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 `Least Recently Used` algorithm-based eviction policy.
```
Comment (daneshk) [inline examples/cache-basics/cache_basics.md:4]:
```suggestion
The `cache` library provides an in-memory cache implementation APIs and uses the `Least Recently Used` algorithm-based eviction policy.
```

File: examples/cache-basics/cache_basics.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
-For more information on the underlying module,
-see the [Cache module](https://docs.central.ballerina.io/ballerina/cache/latest/).
+
+For more information on the underlying module, see the [Cache module](https://docs.central.ballerina.io/ballerina/cache/latest/).
```
Comment (daneshk) [inline examples/cache-basics/cache_basics.md:7]:
```suggestion
For more information on the underlying module, see the [Cache module](https://docs.central.ballerina.io/ballerina/cache/latest/).

This example illustrates basic operations provided by the `cache` library.
```

File: examples/cache-basics/cache_basics.md [hunk 5]
```diff
@@ -1,10 +1,14 @@
 # Cache basics
 
-The Ballerina Cache API provides an in-memory cache implementation by default with a
-`Least Recently Used` algorithm-based eviction policy.
-For more information on the underlying module,
-see the [Cache module](https://lib.ballerina.io/ballerina/cache/latest/).
+The `cache` library provides an in-memory cache implementation APIs and
```
Comment (praneesha) [inline examples/cache-basics/cache_basics.md:3]:
```suggestion
The `cache` library provides in-memory cache implementation APIs and 
```

File: examples/cache-basics/cache_basics.md [hunk 6]
```diff
@@ -1,10 +1,14 @@
...
-The Ballerina Cache API provides an in-memory cache implementation by default with a
-`Least Recently Used` algorithm-based eviction policy.
-For more information on the underlying module,
-see the [Cache module](https://lib.ballerina.io/ballerina/cache/latest/).
+The `cache` library provides an in-memory cache implementation APIs and 
+uses the `Least Recently Used` algorithm-based eviction policy.
+
+For more information on the underlying module, see the [`cache` module](https://lib.ballerina.io/ballerina/cache/latest/).
+
+This example illustrates basic operations provided by the `cache` library.
```
Comment (praneesha) [inline examples/cache-basics/cache_basics.md:8]:
```suggestion
This example illustrates the basic operations provided by the `cache` library.
```

File: examples/cache-basics/cache_basics.md [hunk 7]
```diff
@@ -1,10 +1,14 @@
...
 
 ::: code cache_basics.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/cache-basics/cache_basics.md:12]:
```suggestion
To run this sample, use the `bal run` command.
```
The command should be added in the `.out` file with the $sign. Please check and fix all the occurrences.

File: examples/cache-invalidation/cache_invalidation.md [hunk 8]
```diff
@@ -1,10 +1,12 @@
 # Cache invalidation
 
-The Ballerina Cache API provides an in-memory cache implementation by default with the
-`Least Recently Used` algorithm-based eviction policy.
-For more information on the underlying module, 
-see the [Cache module](https://lib.ballerina.io/ballerina/cache/latest/).
+The `cache` library provides an in-memory cache implementation APIs and
```
Comment (praneesha) [inline examples/cache-invalidation/cache_invalidation.md:3]:
```suggestion
The `cache` library provides in-memory cache implementation APIs and
```

File: examples/cache-invalidation/cache_invalidation.md [hunk 9]
```diff
@@ -1,10 +1,12 @@
...
+uses the `Least Recently Used` algorithm-based eviction policy.
+
+For more information on the underlying module, see the [`cache` module](https://lib.ballerina.io/ballerina/cache/latest/).
 
 ::: code cache_invalidation.bal :::
 
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/cache-invalidation/cache_invalidation.md:10]:
```suggestion
To run this sample use the `bal run` command.
```
Same as above.

File: examples/directories/directories.md [hunk 10]
```diff
@@ -1,9 +1,11 @@
 # Directories
 
-The Ballerina File API contains functions to perform directory operations.<br/><br/>
-For more information on the underlying module, 
-see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
+The `file` library provides APIs to perform directory operations.
+
+For more information on the underlying module, see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
```
Comment (praneesha) [inline examples/directories/directories.md:5]:
```suggestion
For more information on the underlying module, see the [`file` module](https://lib.ballerina.io/ballerina/file/latest/).
```

File: examples/directories/directories.md [hunk 11]
```diff
@@ -1,9 +1,11 @@
...
 
 ::: code directories.bal :::
 
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/directories/directories.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/directory-listener/directory_listener.md [hunk 12]
```diff
@@ -1,10 +1,15 @@
...
-The `Directory Listener` is used to listen to a directory in the local file system. 
-It notifies when new files are created in the directory or when the existing files are deleted or modified.<br/><br/>
-For more information on the underlying module, 
-see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
+The `file` library provides a `Directory Listener`, which is used to listen to the given directory in the local file system.
+It notifies when new files are created in the directory or when the existing files are deleted or modified.
+
+For more information on the underlying module, see the [`file` module](https://lib.ballerina.io/ballerina/file/latest/).
+
+Before running this, change the value of the 'path' field to indicate the path of the directory that you want to monitor.
```
Comment (praneesha) [inline examples/directory-listener/directory_listener.md:8]:
```suggestion
Before running this, change the value of the `path` field to indicate the path of the directory that you want to monitor. 
```

File: examples/directory-listener/directory_listener.md [hunk 13]
```diff
@@ -1,10 +1,15 @@
...
+It notifies when new files are created in the directory or when the existing files are deleted or modified.
+
+For more information on the underlying module, see the [`file` module](https://lib.ballerina.io/ballerina/file/latest/).
+
+Before running this, change the value of the 'path' field to indicate the path of the directory that you want to monitor. 
+As the recursive property is set to false, the listener does not monitor the child directories of the main directory that it listens to.
 
 ::: code directory_listener.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/directory-listener/directory_listener.md:13]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above. The instruction should go to the MD file.

File: examples/filepaths/filepaths.md [hunk 14]
```diff
@@ -1,10 +1,11 @@
...
-target operating system.<br/><br/>
-For more information on the underlying module, 
-see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
+The `file` library also provides APIs to manipulate file paths in a way that is compatible with the target operating system.
+
+For more information on the underlying module, see the [`file` module](https://lib.ballerina.io/ballerina/file/latest/).
 
 ::: code filepaths.bal :::
 
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/filepaths/filepaths.md:9]:
```suggestion
To run this sample use the `bal run` command.
```
Same as above.

File: examples/files/files.md [hunk 15]
```diff
@@ -1,9 +1,11 @@
...
-The Ballerina File API contains functions to perform file-system operations.<br/><br/>
-For more information on the underlying module, 
-see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
+The `file` library provides APIs to perform file-system operations.
+
+For more information on the underlying module, see the [`file` module](https://lib.ballerina.io/ballerina/file/latest/).
 
 ::: code files.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/files/files.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/manage-scheduled-jobs/manage_scheduled_jobs.md [hunk 16]
```diff
@@ -1,10 +1,11 @@
 # Manage sheduled jobs
 
-The `task` library provides functions to manage the scheduled jobs such as pause, resume,
-unschedule, and etc.<br/><br/>
-For more information on the underlying module,
-see the [Regex module](https://lib.ballerina.io/ballerina/regex/latest/).
+The `task` library provides functions to manage the scheduled jobs such as pause, resume, unschedule, and etc.
```
Comment (praneesha) [inline examples/manage-scheduled-jobs/manage_scheduled_jobs.md:3]:
```suggestion
The `task` library provides functions to manage the scheduled jobs such as pause, resume, unscheduled, etc.
```

File: examples/manage-scheduled-jobs/manage_scheduled_jobs.md [hunk 17]
```diff
@@ -1,10 +1,11 @@
...
+
+For more information on the underlying module, see the [`Task` module](https://lib.ballerina.io/ballerina/task/latest/).
```
Comment (praneesha) [inline examples/manage-scheduled-jobs/manage_scheduled_jobs.md:5]:
```suggestion
For more information on the underlying module, see the [`task` module](https://lib.ballerina.io/ballerina/task/latest/).
```

File: examples/manage-scheduled-jobs/manage_scheduled_jobs.md [hunk 18]
```diff
@@ -1,10 +1,11 @@
...
 
 ::: code manage_scheduled_jobs.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/manage-scheduled-jobs/manage_scheduled_jobs.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/regular-expressions/regular_expressions.md [hunk 19]
```diff
@@ -1,10 +1,12 @@
 # Regular expressions
 
 The `regex` library provides functionalities for searching, splitting, and replacing
-the set of characters of the string by using the `regular expression`.<br/><br/>
-For more information on the underlying module,
-see the [Regex module](https://lib.ballerina.io/ballerina/regex/latest/).
+the set of characters of the string by using the `regular expression`.
+
+For more information on the underlying module, see the [`Regex` module](https://lib.ballerina.io/ballerina/regex/latest/).
```
Comment (praneesha) [inline examples/regular-expressions/regular_expressions.md:6]:
```suggestion
For more information on the underlying module, see the [`regex` module](https://lib.ballerina.io/ballerina/regex/latest/).
```

File: examples/regular-expressions/regular_expressions.md [hunk 20]
```diff
@@ -1,10 +1,12 @@
...
 
 ::: code regular_expressions.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/regular-expressions/regular_expressions.md:10]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/task-frequency-job-execution/task_frequency_job_execution.md [hunk 21]
```diff
@@ -1,11 +1,11 @@
...
-end time, and maximum count.
-For more information on the underlying module, 
-see the [Task module](https://lib.ballerina.io/ballerina/task/latest/).
+The `task` library provides an API to schedule Ballerina jobs periodically.
+
+For more information on the underlying module, see the [`task` module](https://lib.ballerina.io/ballerina/task/latest/).
 
 ::: code task_frequency_job_execution.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/task-one-time-job-execution/task_one_time_job_execution.md [hunk 22]
```diff
@@ -1,10 +1,11 @@
...
-The `task:Job` and time should be specified to schedule the job.
-For more information on the underlying module, 
-see the [Task module](https://lib.ballerina.io/ballerina/task/latest/).
+The `task` library provides an API to schedule Ballerina jobs at a specific time.
+
+For more information on the underlying module, see the [`task` module](https://lib.ballerina.io/ballerina/task/latest/).
 
 ::: code task_one_time_job_execution.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/task-one-time-job-execution/task_one_time_job_execution.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/temp-files-directories/temp_files_directories.md [hunk 23]
```diff
@@ -1,9 +1,11 @@
 # Temp files and directories
 
-The Ballerina File API contains functions to perform temp file and directory operations.<br/><br/>
-For more information on the underlying module, 
-see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
+The `file` provides APIs to perform temp file and directory operations.
+
+For more information on the underlying module, see the [File module](https://lib.ballerina.io/ballerina/file/latest/).
```
Comment (praneesha) [inline examples/temp-files-directories/temp_files_directories.md:5]:
```suggestion
For more information on the underlying module, see the [`file` module](https://lib.ballerina.io/ballerina/file/latest/).
```

File: examples/temp-files-directories/temp_files_directories.md [hunk 24]
```diff
@@ -1,9 +1,11 @@
...
 
 ::: code temp_files_directories.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/temp-files-directories/temp_files_directories.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/xslt-transformation/xslt_transformation.md [hunk 25]
```diff
@@ -1,9 +1,11 @@
 # XSLT transformation
 
-The Ballerina `xslt` library contains an API to transform XML content to HTML using XSL transformation.
-For more information on the underlying module,
-see the [XSLT module](https://lib.ballerina.io/ballerina/xslt/latest/).
+The `xslt` library provides an API to transform XML content to HTML using XSL transformation.
+
+For more information on the underlying module, see the [XSLT module](https://docs.central.ballerina.io/ballerina/xslt/latest/).
```
Comment (praneesha) [inline examples/xslt-transformation/xslt_transformation.md:5]:
```suggestion
For more information on the underlying module, see the [`xslt` module](https://docs.central.ballerina.io/ballerina/xslt/latest/).
```

File: examples/xslt-transformation/xslt_transformation.md [hunk 26]
```diff
@@ -1,9 +1,11 @@
...
 
 ::: code xslt_transformation.bal :::
 
+To run this sample, use the `bal run` command.
```
Comment (praneesha) [inline examples/xslt-transformation/xslt_transformation.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```
Same as above.

File: examples/cache-basics/cache_basics.md (hunk 7 above)
Comment (kalaiyarasiganeshalingam) [reply examples/cache-basics/cache_basics.md:12]: All the commands added in the `.out` file with the $sign

**Review by praneesha — APPROVED** [review]

**Pull request merged**